cernopendata_client/cli.py

```diff
@@ -243,6 +243,13 @@ def get_file_locations(server, recid, doi, title, protocol, expand, verbose):
         DOWNLOAD_RETRY_SLEEP
     ),
 )
+@click.option(
+    "--download-engine",
+    "download_engine",
+    default="requests",
+    type=click.Choice(["requests", "pycurl"]),
+    help="Library to be used in downloading files [requests,pycurl, xrootd] [default=requests]",
```

Comment:
Please replace "Library by "Engine" here and elsewhere, since the option is called "engine".